[Armadillo] build failure

**Host Environment**
- OS: Windows 10
- Compiler: MVSC v142

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install armadillo:x64-windows`
Within CMakeLists.txt:
`find_package(Armadillo CONFIG REQUIRED)`
`target_link_libraries(main PRIVATE ${ARMADILLO_LIBRARIES})`

**Failure logs** 

```CMake Error at C:/dev/vcpkg/vcpkg/installed/x64-windows/share/Armadillo/ArmadilloConfig.cmake:12 (include):
  include could not find requested file:

    /share/Armadillo/ArmadilloLibraryDepends.cmake		
C:/dev/vcpkg/vcpkg/installed/x64-windows/share/Armadillo/ArmadilloConfig.cmake	12	
````

Upon looking at the above folder, the file that it is unable to find is in fact there.

**Additional context**

This is the same error as https://github.com/microsoft/vcpkg/issues/8480#issue-503571107

Listed armadillo version on sourceforge is ahead of the vcpkg version: 
http://arma.sourceforge.net/download.html
https://vcpkg.info/port/armadillo

--- a/ports/armadillo/portfile.cmake
+++ b/ports/armadillo/portfile.cmake
@@ -27,7 +27,7 @@ vcpkg_cmake_configure(
 )
 
 vcpkg_cmake_install()
-vcpkg_cmake_config_fixup(CONFIG_PATH share/Armadillo/CMake)
+vcpkg_cmake_config_fixup(PACKAGE_NAME Armadillo CONFIG_PATH share/Armadillo/CMake)
 
 vcpkg_copy_pdbs()
 

--- a/ports/armadillo/vcpkg.json
+++ b/ports/armadillo/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "armadillo",
   "version": "10.6.2",
+  "port-version": 1,
   "description": "Armadillo is a high quality linear algebra library (matrix maths) for the C++ language, aiming towards a good balance between speed and ease of use",
   "homepage": "http://arma.sourceforge.net",
   "dependencies": [

--- a/versions/a-/armadillo.json
+++ b/versions/a-/armadillo.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "34c37cb8151d003da697e9d725033567a4ccb2f7",
+      "version": "10.6.2",
+      "port-version": 1
+    },
+    {
       "git-tree": "66b31fe832e9322876f427f1fec7836bb71c082c",
       "version": "10.6.2",
       "port-version": 0

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -158,7 +158,7 @@
     },
     "armadillo": {
       "baseline": "10.6.2",
-      "port-version": 0
+      "port-version": 1
     },
     "arrayfire": {
       "baseline": "3.7.3",

